apps/transifex/.env.staging

```diff
@@ -0,0 +1,3 @@
+VITE_BACKEND_HOST=https://contentfulapp.stg.transifex.net
+VITE_CONTENTFUL_APP_HOST=https://contentfulapp.stg.transifex.net
+VITE_TXC_HOST=https://app.transifex.com
```

Comment:
At this point we don't have an official staging environment to host this, so I am not sure it is necessary to have staging references in this code at this time. But it is up to you. We plan on implementing an official MPA Staging environment where we will can then create a new app definition ID for this to deploy to.